utils/R/utils.R

```diff
@@ -665,6 +665,25 @@ convert.expr <- function(expression) {
   return(list(variable.drv = deri.var, variable.eqn = list(variables = variables, expression = deri.eqn)))
 }
 
+##' Simple function to use ncftpget for FTP downloads behind a firewall.
+##' Requires ncftpget and a properly formatted config file in the users 
+##' home directory
+##' @title download.file
+##' @param url complete URL for file download
+##' @param destfile destination file name
+##' @param method Method of file retrieval.  Currently the only option is ncftpget
+##' @export
+##' @author Shawn Serbin
+download.file <- function(url, destfile, method="ncftpget") {
+   if (method=="ncftpget") {
```

Comment:
leave method blank, then add `method <- if (missing(method)) getOption("download.file.method", default = "auto")` in the else we can call utils::download.file(ftplink, output_zip_file). We might want to add a special `download.ftp.method` so we can switch to that if it starts with ftp://